portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,175 @@
   width: 650px;
 }
 
-#greeting-container {
+#body {
+     background-image: linear-gradient(to right,rgba(64, 95, 151, 0.8), rgba(255,0,0,0), rgba(64, 95, 151, 0.8));
+  }
+  #content {
+    border-radius: 10px;
+    border: 10px solid black;
+    width: 90%;
+    background-color:whitesmoke;
```

Comment:
fyi: best practice is to not used these named colors as they may appear slightly differently in different browsers. Usually if I find a color I like I figure out what its hex code is and use that.